### DISCUSSION TITLE
Help Please

### DISCUSSION THREAD

**Discussion by Acebull69** [SpiralLinux/SpiralLinux-project#190] [General]
G'Day All  from Australia and the name is Ted
 I hope someone can help I just got a [DELL Vostro 3750 Laptop 17.3" i7-2630QM 6GBRAM 240GBSSD and I would like to know if I would have any problems installing Spiral as the Del is about 11 years old 

                                                                    Thank you Ted

**Comment by artytux** [comment]
Hey @Acebull69

With the hardware just looking at from i7-2630QM 6GBRAM 240GBSSD  about 11 years old seems it should have enough grunt to do most duties except maybe hi-end graphics work, enough ram yeah more than enough disk space 240GB especially if you are storing your stuff (music, wallpapers and documents on an external hard-drive also helps to have an extra hard drive for the backups.

It is not SpiralLinux that will brick itself its the hardware that can ! Yours is older so be safe use Backups.
Did you verify your downloaded ISO ??
How about the live session off an USB, How did that go, any problems, did you try most of the apps should be able to connect to Ext HD on Live Session yeah, to find out if LibreOffice Writer works OK and Kate, I'm bias to KDE, try playing music in Live Session mode too.

That should give an indication of No was all downhill or Yes everything I tried worked well Inc WWW connection.

I ask because you don't tell us what you tried maybe your new to Linux IF SO try the things suggested and that will give a very good starting point.
Also you never mentioned which desktop  you are thinking of installing.
May I suggest more info will get you better advice from other users but that's OK if your new to Linux
or take off the tin foil hat please.
Makes mind-reading difficult.


Wish you fun

Best Regards
artytux

**Reply by Acebull69** [reply]
G'Day artytux  And thank you for your comments The Dell still has Window 10 on it and I would like to put Spirallinux on it . But I have read online that some people in the past have had trouble putting Linux on . This model  of DELL Vostro 3750 Laptop but that was a long time ago . Some my question is do you think that Spirallinux  will work ok on it as I don"t want Window 10 on it as I have not used Window for close to 10 years I just love Linux hope you can Help 

                                                                     Cheers Ted

**Comment by artytux** [comment]
Hello
@Acebull69 Since your going to wipe clean the machine,

This is not the only machine you've got,
Does it have a Nvidia card that would be maybe the hurdle with drivers although SpiralLinux with the Debian repos may be helpful,
I don't use Nvidia so someone else will have to chime in for that ,
again run into a problem start a new thread about the Nvidia card problems and black screen on startup, all able to be solved.
If it can be solved in Buntu and Mint it **can be ** solved here.
Since you don't use Window$, same for me too.
I presume you know about partitoning, will you use Ext4 or Btrfs (is that right  Btrfs ?, sorry), I prefer ext4 so i can have BackInTime and TimeShift, easy peasy.
Things I've noticed all these people having problems does not mean you **will have** it just means the potential is there,
just because a few people and it is only a few, have problems installing their Linux, they are on Mint and Buntu same damn fishtank,
Look up DDG  "Install Linux on Vostro 3750" either copy page  >Firefox > Print or scribble some notes before the install and if you run into any problems use the safe mode and TTY,
Since they are on Mint and Buntu, never presume their problems will happen to you.
SpiralLinux is a clean Debian and completely different to the others so chances are those problems **may not appear**

And if any problems arise start a new thread, if you have use Linux for a while now you probably also know the Linux forums are always there with help.

Best Regards
artytux

**Reply by Acebull69** [reply]
G'Day artytux  Well installed Spirallinux  and all seems ok so all good for me so Thank You  

                                                                Cheers  Ted
P>S But I will get back to the Forum if I have any problems

**Comment by artytux** [comment]
Hello @Acebull69
I'm happy that the install went pa-ding GOOD
Now you should realize that any problems arising in the future will probably because you did some weird thing ( machines who knew ? Bill !)
Soo feel free to tell us how you DID whAT , that's how we learn, it is fun,
If you don't break things how can you learn to fix things
There are many many smart Linux users in the background 0Oh they are there ! ! ! 
So if you got that other one BRTFX ? Nah hang on brtfs or ext4 MAKE sure you HAVE backup and test the backup after a few days
After a long session of altering files or end of day have your backup enabled !
An untested backup is not viable !
it's the one of the things that supports your computering machine for the next DAY.
Unless you prefer fresh install, joke seriously .

All the Best 
Regards
artytux
